ingeniamotion/metaclass.py

```diff
@@ -45,7 +45,7 @@ def check_servo(cls, func: Callable[..., T]) -> Callable[..., T]:
         """
 
         @wraps(func)
-        def wrapper(self, *args, **kwargs):  # type: ignore
+        def wrapper(self, *args, **kwargs):  # type: ignore[no-untyped-def]
             mc = self.mc
             func_args = inspect.getfullargspec(func).args
```

Comment:
Unrelated to this, but when reading I realized that this might be a performance issue: INGM-579